daikon-spring/daikon-spring-audit-logs/src/test/java/org/talend/daikon/spring/audit/logs/api/AuditLogTest.java

```diff
@@ -46,11 +47,31 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = AuditLogTestApp.class)
 @AutoConfigureMockMvc
-@TestPropertySource(properties = { "audit.enabled=true", "spring.application.name=daikon",
-        "audit.kafka.bootstrapServers=localhost:9092" })
+@TestPropertySource(properties = { //
+        "audit.enabled=true", //
+        "spring.application.name=daikon", //
+        "audit.kafka.bootstrapServers=localhost:9092", //
+        "audit.trusted-proxies=" + TRUSTED_PROXIES //
+})
 @Import(AuditLogTestConfig.class)
 public class AuditLogTest {
 
+    private static final String REMOTE_IP_HEADER = "X-Forwarded-For";
+
+    public static final String TRUSTED_PROXIES = "42.42.42.42";
+
+    private static final String MY_IP = "35.74.154.242";
+
+    private static final String MY_IP_WITH_INVALID_IP = "35.74.154.242, ImAWrongIp";
+
+    private static final String MY_IP_WITH_INTERNAL_PROXIES = "35.74.154.242, 10.72.5.245, 10.80.17.172";
+
+    private static final String MY_IP_WITH_INTERNAL_PROXIES_AND_TRUSTED_PROXIES = "35.74.154.242, 42.42.42.42, 10.72.5.245, 10.80.17.172";
+
+    private static final String MY_IP_WITH_FORGERY_ATTEMPT = "51.51.51.51, 35.74.154.242";
```

Comment:
if it was a forgery attemp the IP should reversed
```suggestion
    private static final String MY_IP_WITH_FORGERY_ATTEMPT = "35.74.154.242, 51.51.51.51";
```